modules/systemd-journal/journal-reader.c

```diff
@@ -584,6 +584,39 @@ _add_poll_events(JournalReader *self)
   return TRUE;
 }
 
+static gint
+_journal_open(JournalReader *self)
+{
+#if SYSLOG_NG_HAVE_JOURNAL_NAMESPACES
+  gint journal_flags = SD_JOURNAL_LOCAL_ONLY;
+  gchar *journal_namespace;
```

Comment:
```suggestion
  gchar *journal_namespace = NULL;
```

If the `self->options->namespace == "*"` than we use journal_namespace as an argument in https://github.com/syslog-ng/syslog-ng/pull/3358/files#diff-8506b8af7604a75a9a5108a18b1477dcR613 without initialising it's value. Giving `NULL` as a namespace string is acceptable together with the `SD_JOURNAL_ALL_NAMESPACES` flag, so the code is good, only the variable is not initialised properly.

Reply:
Thx, I've modified the squashed commit accordingly.